themes/custom/az_barrio/az_barrio.theme

```diff
@@ -446,5 +450,18 @@ function az_barrio_preprocess_image(&$variables) {
     unset($variables['attributes']['height']);
     $variables['attributes']['class'] = ['card-img-top'];
   }
+
+}
+
+function az_barrio_preprocess_block(&$variables) {
+  $blockid = str_replace(array('block-', '-'), array('', '_'), $variables['attributes']['id']);
+  $variables['content']['#attributes']['block'] = $blockid;
+ }
+
+function az_barrio_theme_suggestions_menu_alter(array &$suggestions, array $variables) {
+  if (isset($variables['attributes']['block'])) {
+    \Drupal::logger('majors_data')->warning(print_r($suggestions, TRUE));
```

Comment:
Don't we want this to be `az_barrio` instead of `majors_data`?
```suggestion
    \Drupal::logger('az_barrio')->warning(print_r($suggestions, TRUE));
```